en/04_Changelogs/beta/5.0.0-beta1.md

```diff
@@ -11,7 +76,81 @@ We will only consider merging post-beta breaking API changes if they are necessa
+- silverstripe/graphql [![CI](https://github.com/silverstripe/silverstripe-graphql/actions/workflows/ci.yml/badge.svg?branch=5)](https://github.com/silverstripe/silverstripe-graphql/actions/workflows/ci.yml)
+- silverstripe/versionned-admin [![CI](https://github.com/silverstripe/silverstripe-versioned-admin/actions/workflows/ci.yml/badge.svg?branch=2)](https://github.com/silverstripe/silverstripe-versioned-admin/actions/workflows/ci.yml)
+- silverstripe/campaign-admin [![CI](https://github.com/silverstripe/silverstripe-campaign-admin/actions/workflows/ci.yml/badge.svg?branch=2)](https://github.com/silverstripe/silverstripe-campaign-admin/actions/workflows/ci.yml)
+- silverstripe/cms [![CI](https://github.com/silverstripe/silverstripe-cms/actions/workflows/ci.yml/badge.svg?branch=5)](https://github.com/silverstripe/silverstripe-cms/actions/workflows/ci.yml)
+- silverstripe/session-manager [![CI](https://github.com/silverstripe/silverstripe-session-manager/actions/workflows/ci.yml/badge.svg?branch=2)](https://github.com/silverstripe/silverstripe-session-manager/actions/workflows/ci.yml)
+- silverstripe/recipe-core [![CI](https://github.com/silverstripe/recipe-core/actions/workflows/ci.yml/badge.svg?branch=5)](https://github.com/silverstripe/recipe-core/actions/workflows/ci.yml)
+- silverstripe/staticpublishqueue [![CI](https://github.com/silverstripe/silverstripe-registry/actions/workflows/ci.yml/badge.svg?branch=3)](https://github.com/silverstripe/silverstripe-registry/actions/workflows/ci.yml)
+
+#### Other issues
+
+The CMS 5 compatible version of `tractorcow/silverstripe-fluent` is temporarily being released under `silverstripe/silverstripe-fluent`.
+
+View the [Silverstripe CMS 5 compatibility](https://github.com/tractorcow-farm/silverstripe-fluent/issues/776) issue on the Fluent repository for progress.
+
+## Upgrading your third party module
+
+One of the primary purposes of the beta testing phase is to give third party maintainers and developers an opportunity to upgrade their modules to be compatible with Silverstripe CMS 5.
+
+### Planning your upgrade
+
+You should decide if you intend to do a new major release of your module to add support for Silverstripe CMS 5.
+
+Most of the key Silverstripe CMS APIs have not changed between CMS 4 and CMS 5. That said, take time to read this changelog to identify changes that might be problematic for your project.
+
+Depending on how complex your module is, it may be completely practical to create a version that works with both Silverstripe CMS 4 and Silverstripe CMS 5. We do not necessarily recommend this approach. It may require you to tag a major release later on if you want to make use of new Silverstripe CMS 5 feature later on. But you may wish to consider this approach to minimise your maintenance burden.
```

Comment:
```suggestion
Depending on how complex your module is, it may be completely practical to create a version that works with both Silverstripe CMS 4 and Silverstripe CMS 5. We do not necessarily recommend this approach as it may require you to tag a major release later on if you want to make use of new Silverstripe CMS 5 features later on. But you may wish to consider this approach to minimise your maintenance burden.
```